.github/workflows/haskell.yml

```diff
@@ -0,0 +1,110 @@
+name: Haskell CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    defaults:
+      run:
+        shell: bash
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ghc: ["8.6.5"]
```

Comment:
No `8.10.2` ?

Reply:
`8.10.2` will need to wait until the project updates `cardano-node` dependency to a version that supports this compiler.